inspektor/reindent.py

```diff
@@ -104,14 +110,14 @@ def run(self):
                     want = have2want.get(have, -1)
                     if want < 0:
                         # Then it probably belongs to the next real stmt.
-                        for j in xrange(i + 1, len(stats) - 1):
+                        for j in range(i + 1, len(stats) - 1):
```

Comment:
Might I ask what is the `range` needed for here? You're just iterating so it should be safe and faster to use `xrange`, shouldn't it?